src/adapter/etl-adapter-xml/src/Flow/ETL/Adapter/XML/Loader/DomDocumentLoader.php

```diff
@@ -17,9 +17,9 @@ final class DomDocumentLoader implements Closure, Loader, Loader\FileLoader
 {
     public function __construct(
         private readonly Path $path,
-        private readonly RowsNormalizer $normalizer = new RowsNormalizer(),
         private readonly string $collectionName = 'rows',
         private readonly string $collectionElementName = 'row',
+        private readonly RowsNormalizer $normalizer = new RowsNormalizer(),
```

Comment:
honestly I dont think we want to expose that, I don't see any reason why anyone should touch normalizers.